### PR TITLE
sort by pageid?

### PR DESCRIPTION
**Pull request by fjf2002**
My company had the request to sort by page id, so I made a new flag `sortbypageid`.

Any interest to include this into the master branch?

### CODE REVIEW

**Comment by fjf2002** [comment]
*ping* any news here?

File: helper/syntax.php [hunk 1]
```diff
@@ -237,6 +239,7 @@ public function parseFlags($flags)
             'images' => false,
             'count' => false,
             'tagintersect' => false,
+            'sortbypageid' => false
```
Comment (Klap-in) [inline helper/syntax.php:242]: Please update the phpdocs as well please.

**Review by Klap-in — CHANGES_REQUESTED** [review]
Looks fine, not tested yet.
Could you please fix the minor comment?

**Comment by fjf2002** [comment]
@Klap-in: Thanks for your response.
However I mostly am interested in using the include plugin to render content (instead of links), see PR #27 . Thus pagelist gets bypassed/replaced by the include plugin, and pagelist can't sort for me.

**Comment by fjf2002** [comment]
*ping* any interest to merge this?

**Comment by fjf2002** [comment]
Thanks a lot. Shall I document this at https://www.dokuwiki.org/plugin:tagfilter or wait until the plugin's next release ?

**Comment by peterfromearth** [comment]
yes, would be great if you could update it.

**Comment by fjf2002** [comment]
> yes, would be great if you could update it.

Done. You can decide if you want to mark the pagelist plugin as optional - it should be an either-or of necessity between pagelist plugin an include plugin.

**Pull request merged**